app-typescript/components/IconLabel.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import classNames from 'classnames';
+
+interface IProps {
+    text?: string;
+    color?: string;  //  https://ui-framework.superdesk.org/#/components/colors
```

Comment:
There isn't file with listed available colors. Colors are defined in sass and created with function.

Reply:
Can you link to the sass file where available color strings are listed?

Reply:
It is better to have link to docs page, it is more readable and feature developers will know which color to pick. If we add link to sass file it will mean nothing. Because colors are created with multiple sass functions and they will be changed in future. @gagily leave it as it is.

Reply:
Docs page works as well. Can you then add a link to the docs page?

Reply:
Not that docs page :smile: This is the same you linked before which I commented not to link because Vladimir said it's for internal use only. Also, it doesn't represent the actual values. Here's the example code from the docs you've added and none of the colors you use are specified in that docs page.

```
<IconLabel text='Label primary'  icon='facebook' color='primary'/>
<IconLabel text='Label success' icon='text' color='success'/>
<IconLabel text='Label warning' icon='paywall' color='warning'/>
<IconLabel text='Label alert' icon='trash' color='alert'/>
<IconLabel text='Label highlight' icon='composite' color='highlight'/>
<IconLabel text='Label sd-green' icon='video' color='sd-green'/>
```

Reply:
@tomaskikutis I think you got this wrong. It is completely fine as it is now. I don't know what internal use means in this case, the whole framework is for internal use. 'primary', 'success' etc. are semantic colours with appropriate names, colours listed in the extended colour library don't have any semantic meaning but are in this case needed for differentiation between different labels . As you know labels are in our case very specific and as we use them to represent different things with it.

Reply:
As we can see in the example snippet, the following strings are available as color a parameter for `IconLabel` component - `primary, success, warning, alert, highlight, sd-green`.

My question is, what are all possible color names? Is it only these 6? If so, I want the component interface to say so like `color: 'primary' | 'success' | 'warning' | 'alert' | 'highlight' | 'sd-green'''. Otherwise, I'd expect a link with available color names listed.

Reply:
Primary, Success, Warning, Alert are default colors (similar as bootstrap), and also, if you apply `red-500` it will work.

Reply:
Have you tried it? It doesn't work for me. I use the following code, but the label is gray

```
<IconLabel text='Label' icon='video'  color='red-500'/ >
```

Reply:
I've fixed it now. Also format of colors are 'red--500' with two dashes.